### PR TITLE
Midsummersprint fix transforms

### PR DESCRIPTION
**Pull request by MrTango**

### CODE REVIEW

File: Products/PortalTransforms/tests/test_transforms.py [hunk 1]
```diff
@@ -415,7 +440,7 @@ def make_tests(test_descr=TRANSFORMS_TESTINFO):
             except MissingBinary:
                 # we are not interessted in tests with missing binaries
                 continue
-            except:
+            except:  # noqa
```
Comment (gforcada) [inline Products/PortalTransforms/tests/test_transforms.py]: Rather than a ``# noqa`` you can add ``except Exception:``

**Comment by gforcada** [comment]
Thanks for porting it over to plone.app.testing!!

File: Products/PortalTransforms/tests/test_transforms.py [hunk 2]
```diff
@@ -201,6 +201,20 @@ def setUp(self):
         registry = getUtility(IRegistry)
         self.settings = registry.forInterface(
             IFilterSchema, prefix="plone")
+        self.settings.valid_tags.append('style')
+        import transaction
+        transaction.commit()
```
Comment (gforcada) [inline Products/PortalTransforms/tests/test_transforms.py]: @MrTango if you make manual commits you need to switch from integration to functional layer, if not we will have test isolation problems later on

**Comment by gforcada** [comment]
@MrTango I removed the transaction.commit (at least locally the tests continued to pass without it), so we don't need to move the test to a functional layer.

**Comment by pbauer** [comment]
The upgrade-step for this change is in https://github.com/plone/plone.app.upgrade/commit/60d6c49ac1d7a4dae12f6b8f5eda7c594fc1060a

**Pull request merged**